src/test/java/io/stargate/sgv2/jsonapi/api/v1/IndexingConfigIntegrationTest.java

```diff
@@ -0,0 +1,345 @@
+package io.stargate.sgv2.jsonapi.api.v1;
+
+import static io.restassured.RestAssured.given;
+import static io.stargate.sgv2.common.IntegrationTestUtils.getAuthToken;
+import static org.hamcrest.Matchers.*;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+import io.stargate.sgv2.jsonapi.config.constants.HttpConstants;
+import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
+import org.junit.jupiter.api.*;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(DseTestResource.class)
+@TestClassOrder(ClassOrderer.OrderAnnotation.class)
+public class IndexingConfigIntegrationTest extends AbstractNamespaceIntegrationTestBase {
+
+  private static final String denyOneIndexingCollection = "deny_one_indexing_collection";
+
+  private static final String denyManyIndexingCollection = "deny_many_indexing_collection";
+
+  private static final String denyAllIndexingCollection = "deny_all_indexing_collection";
+
+  private static final String allowOneIndexingCollection = "allow_one_indexing_collection";
+
+  private static final String allowManyIndexingCollection = "allow_many_indexing_collection";
+
```

Comment:
@maheshrajamani According to the [design doc](https://docs.google.com/document/d/1Q4F4XCW71VaPZ1UEfGzLuHu7quw_aOrutDi9in7RWoY/edit), the wildcat character "\*" can only be used with a deny list to deny all fields, its use in an allow list generates an error. So I didn't test "\*" in allow list. However, Your previous [PR](https://github.com/stargate/jsonapi/pull/771) didn't handle this case, should you add the error handling for this?

Reply:
There is a minor change to allow `[ "*" ]` for allow list as well, to provide symmetry.